Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.8
```

```suggestion
FROM python:3.8-slim
```

We can slim down a lot the image size :smiley: 
```
$ docker images | grep corsy
corsy      slim        dcd5fc300c58        About a minute ago   122MB
corsy      latest      2e31a7fbc530        6 minutes ago        891MB
```